api/notification-service.yaml

```
@@ -268,6 +268,15 @@ components:
           type: array
           items:
             type: string
+        customArgs:
+          type: object
+          additionalProperties:
+            type: string
+          description: |
+            Custom arguments to be passed to the mail provider.
```

can we briefly indicate its a key/value pair
```suggestion
            Custom arguments is a key/value pair passed to the mail provider.
```